src/lab-tabs/actions/pickup-lab-request-action.component.tsx

```diff
@@ -0,0 +1,36 @@
+import { OverflowMenuItem } from "@carbon/react";
+import { showModal } from "@openmrs/esm-framework";
+import { Order } from "@openmrs/esm-patient-common-lib";
+import React, { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+
+interface PickLabRequestActionMenuProps {
+  order: Order;
+}
+
+const PickupLabRequestAction: React.FC<PickLabRequestActionMenuProps> = ({
+  order,
+}) => {
+  const { t } = useTranslation();
+  const unSupportedStatuses = ["IN_PROGRESS", "COMPLETED", "DECLINED"];
+
+  const launchModal = useCallback(() => {
+    const dispose = showModal("pickup-lab-request-modal", {
+      closeModal: () => dispose(),
+      order,
+    });
+  }, [order]);
+
+  return (
+    <OverflowMenuItem
+      itemText={t("pickupLabRequest", "Pickup Lab Request")}
+      onClick={launchModal}
+      style={{
+        maxWidth: "100vw",
+      }}
```

Comment:
This style seems to be repeated in multiple files. Can we extract this to the styles file so we can have a single point incase of future changes

Reply:
Well like I said, there lots of residue snippets that don't align with our standard practices, this is one of them. More work is required to harmonise the styling in the entire repo. This was just to get us started